plugins/woocommerce/tests/php/includes/abstracts/class-wc-abstract-order-test.php

```
@@ -170,15 +170,15 @@ public function test_add_product_passes_order_to_wc_get_price_excluding_tax() {
 			)
 		);
 
-		//phpcs:disable Squiz.Commenting
+		// phpcs:disable Squiz.Commenting
 		$order_item = new class() extends WC_Order_Item_Product {
 			public $passed_props;
 
 			public function set_props( $args, $context = 'set' ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$context'. (undefined)

(UnusedFormalParameter)